devices/Sen5x/Entities/VOCAlgorithmState.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+using System;
+
+namespace Iot.Device.Sen5x.Entities
+{
+    /// <summary>
+    /// Represents the VOC algorithm state.
+    /// </summary>
+    public class VocAlgorithmState : AbstractReadWriteEntity
+    {
+        internal override int ByteCount => 12;
+
+        internal override void FromSpanByte(SpanByte data)
+        {
+            State = new byte[8];
```

Comment:
Same here and in the various conversion, a comment on how the format looks like would be very appreciated for maintenance point of view.

Reply:
It feels a bit weird to have to explain the "every 3rd byte is a checksum" even though this is perfectly explained in the datasheet. I don't really document when I use `UInt` vs `Int` either. Since I'm not deviating from the datasheet, it feels redundant to explain. If there's a bug, a developer would have to verify my comments against the official documentation anyway.

I'll add a little note to the abstract methods in the base classes that explain the checksum thing.